src/components/ScholarshipsMadeSimpleSection.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import PropTypes from 'prop-types';
+import ScholarshipsMadeSimpleGrid from './ScholarshipsMadeSimpleGrid';
+
+function TabPanel(props) {
+  const { children, value, index } = props;
+
+  return (
+    <div role="tabpanel" hidden={value !== index}>
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+export default function ScholarshipsMadeSimpleSection() {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <div className="stepperStyle">
```

Comment:
`stepperStyle` is not descriptive of what this is.

Reply:
change it to `<div className="ScholarshipGridTab">`, is that more descriptive?

Reply:
Ignore this it was replaced by -> `<Container style={{ background: 'theme.palette.background.paper' }} >`